index.html

```diff
@@ -442,8 +442,13 @@
         <li>If <var>timeRemaining</var> is negative, set it to 0.</li>
         <li>Return <var>timeRemaining</var>.</li>
       </ol>
-      <p>The <code><dfn>didTimeout</dfn></code> attribute on an `IdleDeadline`
-      object MUST return `true` if the callback was invoked by the <a>invoke idle callback timeout algorithm</a>, and `false` otherwise.</p>
+      <p>Each <a>IdleDeadline</a> has an associated <code><dfn>timeout</dfn>
+      </code>, which is initially false. The <code><dfn>
+      didTimeout</dfn></code> getter MUST return <a>timeout</a>. The <a>invoke
+      idle callback timeout algorithm</a> sets <a>timeout</a> to true to specify
+      that the callback is being executed outside an idle period, due to it
+      having exceeded the timeout property passed as an <a>IdleRequestOptions</a>
+      parameter when the callback was registered.</p>
```

Comment:
This last sentence would be better as a note, to clearly indicate it's informative.